articles/azure-monitor/app/monitor-performance-live-website-now.md

```diff
@@ -26,11 +26,11 @@ Status Monitor is used to instrument a .NET application hosted in IIS either on-
 
 ![Screenshot of App Insights overview graphs containing information on failed requests, server response time, and server requests](./media/monitor-performance-live-website-now/overview-graphs.png)
 
-You have a choice of three routes to apply Application Insights to your .NET web applications:
+You have a choice of two routes to apply Application Insights to your .NET web applications:
 
 * **Build time:** [Add the Application Insights SDK][greenbrown] to your web app code.
 * **Run time:** Instrument your web app on the server, as described below, without rebuilding and redeploying the code.
-* **Both:** Build the SDK into your web app code, and also apply the run-time extensions. Get the best of both options.
+* Note: If you use build time instrumentation, run time instrumention will not work even if it is turned on. As long as the run time instrumentation detected there is Application Insights SDK dependency in your application, it will stop working to avoid duplicated data collection.
```

Comment:
@reyang thanks for contributing to our docs!

If you use the word "Note" in this way in the docs the preferred way is to use:
```
> [!NOTE]
> Your note text here...
```
This will allow it to render as a highlighted note section on the page.

Also two other small fixes:

_As long as run time instrumentation **is** detected there is Application Insights SDK dependency **tracking** in your application. It will stop working to avoid duplicated data collection._

Reply:
@reyang if you can make the changes above I would be happy to `#sign-off` and get these merged.